requirements-dev.txt

```diff
@@ -1,3 +1,5 @@
-click>=7.0
+Click>=7.0
```

Comment:
@bdice Not sure why the change, but just so you know: PyPI (pip) is case insensitive: https://www.python.org/dev/peps/pep-0426/#name

Reply:
When I typed pip install click in the cimmand line it showed me an error. And it resolved when Click was typed instead of click. I guess there was a typo error if pip is case insensitive. I'll change it if you want.